.github/workflows/labels.yml

```diff
@@ -4,30 +4,18 @@ on:
   workflow_dispatch:
     inputs:
       dryrun:
-        description: "Preview changes to labels without editing them"
+        description: "dryrun: Preview changes to labels without editing them (true|false)"
         required: true
         default: "true"
 
 jobs:
-  global-labels:
+  sync:
     runs-on: ubuntu-latest
     steps:
-      - uses: EndBug/label-sync@v2
+      - uses: EndBug/label-sync@v2.1.0
         with:
-          config-file: https://raw.githubusercontent.com/conda/infra/main/.github/labels.yml
-          delete-other-labels: true
-          dry-run: ${{ github.event.inputs.dryrun }}
-
-  local-labels:
-    runs-on: ubuntu-latest
-    steps:
-      - id: has_local
-        uses: andstor/file-existence-action@v1.0.1
-        with:
-          files: .github/labels.yml
-      - uses: EndBug/label-sync@v2
-        if: steps.has_local.outputs.files_exists == 'true'
-        with:
-          config-file: .github/labels.yml
+          config-file: |
+            https://raw.githubusercontent.com/conda/infra/main/.github/labels.yml
+            .github/labels.yml
```

Comment:
Open to input about this, but the files here might be best written out as a list?

```
          config-file:
            - https://raw.githubusercontent.com/conda/infra/main/.github/labels.yml
            - .github/labels.yml
```

Reply:
Not possible, all `with` arguments are actually passed to the action as environment variables, e.g. `config-file` is accessible as the env var `INPUT_CONFIG-FILE`. So the value here must be a number or string, using a multiline string and newline as the delimiter is a standard throughout GitHub Actions when passing a list of values.

Reply:
And here's the associated docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith

Reply:
Makes sense, thanks for the clarification! 👍